src/scripts/main.js

```diff
@@ -1,3 +1,102 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.employees-table');
+  const headers = table.querySelectorAll('th');
+  let sortOrder = {}; // Store sort state per column
```

Comment:
The `sortOrder` object is being reset every time a header is clicked. This means that only the last clicked column's sort order is stored. If you want to maintain the sort order for multiple columns, consider updating the `sortOrder` object instead of resetting it.